[CI] MixedClusterClientYamlTestSuiteIT test {p0=search.vectors/90_sparse_vector/Sparse vector in 8.x} failing

Looks like a genuine problem, the error message is unexpected

**Build scan:**
https://gradle-enterprise.elastic.co/s/pbkfjyxg2h7oa/tests/:qa:mixed-cluster:v8.10.0%23mixedClusterTest/org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT/test%20%7Bp0=search.vectors%2F90_sparse_vector%2FSparse%20vector%20in%208.x%7D

**Reproduction line:**
```
./gradlew ':qa:mixed-cluster:v8.10.0#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=search.vectors/90_sparse_vector/Sparse vector in 8.x}" -Dtests.seed=7619629EB75D0707 -Dtests.bwc=true -Dtests.locale=es-HN -Dtests.timezone=CTT -Druntime.java=20
```

**Applicable branches:**
main

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT&tests.test=test%20%7Bp0%3Dsearch.vectors/90_sparse_vector/Sparse%20vector%20in%208.x%7D

**Failure excerpt:**
```
java.lang.AssertionError: Failure at [search.vectors/90_sparse_vector:127]: the error message was expected to match the provided regex but didn't
Expected: The \[sparse_vector\] field type is no longer supported
     but: was "{root_cause=[{type=mapper_parsing_exception, reason=Failed to parse mapping: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions., stack_trace=org.elasticsearch.index.mapper.MapperParsingException: Failed to parse mapping: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:400)\n\tat org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.updateIndexMappingsAndBuildSortOrder(MetadataCreateIndexService.java:1339)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.lambda$applyCreateIndexWithTemporaryService$2(MetadataCreateIndexService.java:472)\n\tat org.elasticsearch.indices.IndicesService.withTempIndexService(IndicesService.java:731)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexWithTemporaryService(MetadataCreateIndexService.java:470)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequestWithV1Templates(MetadataCreateIndexService.java:593)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequest(MetadataCreateIndexService.java:420)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService$1.execute(MetadataCreateIndexService.java:299)\n\tat org.elasticsearch.cluster.service.MasterService$UnbatchedExecutor.execute(MasterService.java:550)\n\tat org.elasticsearch.cluster.service.MasterService.innerExecuteTasks(MasterService.java:1039)\n\tat org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:1004)\n\tat org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:232)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.lambda$run$2(MasterService.java:1626)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.run(MasterService.java:1623)\n\tat org.elasticsearch.cluster.service.MasterService$5.lambda$doRun$0(MasterService.java:1237)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$5.doRun(MasterService.java:1216)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalArgumentException: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.vectors.SparseVectorFieldMapper.lambda$static$0(SparseVectorFieldMapper.java:78)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1483)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1441)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseProperties(ObjectMapper.java:325)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseObjectOrDocumentTypeProperties(ObjectMapper.java:235)\n\tat org.elasticsearch.index.mapper.RootObjectMapper.parse(RootObjectMapper.java:411)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:103)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:97)\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:398)\n\t... 23 more\n}], type=mapper_parsing_exception, reason=Failed to parse mapping: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions., caused_by={type=illegal_argument_exception, reason=The [sparse_vector] field type is not supported from 8.0 to 8.10 versions., stack_trace=java.lang.IllegalArgumentException: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.vectors.SparseVectorFieldMapper.lambda$static$0(SparseVectorFieldMapper.java:78)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1483)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1441)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseProperties(ObjectMapper.java:325)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseObjectOrDocumentTypeProperties(ObjectMapper.java:235)\n\tat org.elasticsearch.index.mapper.RootObjectMapper.parse(RootObjectMapper.java:411)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:103)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:97)\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:398)\n\tat org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.updateIndexMappingsAndBuildSortOrder(MetadataCreateIndexService.java:1339)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.lambda$applyCreateIndexWithTemporaryService$2(MetadataCreateIndexService.java:472)\n\tat org.elasticsearch.indices.IndicesService.withTempIndexService(IndicesService.java:731)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexWithTemporaryService(MetadataCreateIndexService.java:470)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequestWithV1Templates(MetadataCreateIndexService.java:593)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequest(MetadataCreateIndexService.java:420)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService$1.execute(MetadataCreateIndexService.java:299)\n\tat org.elasticsearch.cluster.service.MasterService$UnbatchedExecutor.execute(MasterService.java:550)\n\tat org.elasticsearch.cluster.service.MasterService.innerExecuteTasks(MasterService.java:1039)\n\tat org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:1004)\n\tat org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:232)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.lambda$run$2(MasterService.java:1626)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.run(MasterService.java:1623)\n\tat org.elasticsearch.cluster.service.MasterService$5.lambda$doRun$0(MasterService.java:1237)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$5.doRun(MasterService.java:1216)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1623)\n}, stack_trace=org.elasticsearch.index.mapper.MapperParsingException: Failed to parse mapping: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:400)\n\tat org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.updateIndexMappingsAndBuildSortOrder(MetadataCreateIndexService.java:1339)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.lambda$applyCreateIndexWithTemporaryService$2(MetadataCreateIndexService.java:472)\n\tat org.elasticsearch.indices.IndicesService.withTempIndexService(IndicesService.java:731)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexWithTemporaryService(MetadataCreateIndexService.java:470)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequestWithV1Templates(MetadataCreateIndexService.java:593)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequest(MetadataCreateIndexService.java:420)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService$1.execute(MetadataCreateIndexService.java:299)\n\tat org.elasticsearch.cluster.service.MasterService$UnbatchedExecutor.execute(MasterService.java:550)\n\tat org.elasticsearch.cluster.service.MasterService.innerExecuteTasks(MasterService.java:1039)\n\tat org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:1004)\n\tat org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:232)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.lambda$run$2(MasterService.java:1626)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.run(MasterService.java:1623)\n\tat org.elasticsearch.cluster.service.MasterService$5.lambda$doRun$0(MasterService.java:1237)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$5.doRun(MasterService.java:1216)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalArgumentException: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.vectors.SparseVectorFieldMapper.lambda$static$0(SparseVectorFieldMapper.java:78)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1483)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1441)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseProperties(ObjectMapper.java:325)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseObjectOrDocumentTypeProperties(ObjectMapper.java:235)\n\tat org.elasticsearch.index.mapper.RootObjectMapper.parse(RootObjectMapper.java:411)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:103)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:97)\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:398)\n\t... 23 more\n}"

  at __randomizedtesting.SeedInfo.seed([7619629EB75D0707:FE4D5D4419A16AFF]:0)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:582)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:534)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1623)

  Caused by: java.lang.AssertionError: the error message was expected to match the provided regex but didn't
  Expected: The \[sparse_vector\] field type is no longer supported
       but: was "{root_cause=[{type=mapper_parsing_exception, reason=Failed to parse mapping: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions., stack_trace=org.elasticsearch.index.mapper.MapperParsingException: Failed to parse mapping: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:400)\n\tat org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.updateIndexMappingsAndBuildSortOrder(MetadataCreateIndexService.java:1339)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.lambda$applyCreateIndexWithTemporaryService$2(MetadataCreateIndexService.java:472)\n\tat org.elasticsearch.indices.IndicesService.withTempIndexService(IndicesService.java:731)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexWithTemporaryService(MetadataCreateIndexService.java:470)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequestWithV1Templates(MetadataCreateIndexService.java:593)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequest(MetadataCreateIndexService.java:420)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService$1.execute(MetadataCreateIndexService.java:299)\n\tat org.elasticsearch.cluster.service.MasterService$UnbatchedExecutor.execute(MasterService.java:550)\n\tat org.elasticsearch.cluster.service.MasterService.innerExecuteTasks(MasterService.java:1039)\n\tat org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:1004)\n\tat org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:232)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.lambda$run$2(MasterService.java:1626)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.run(MasterService.java:1623)\n\tat org.elasticsearch.cluster.service.MasterService$5.lambda$doRun$0(MasterService.java:1237)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$5.doRun(MasterService.java:1216)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalArgumentException: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.vectors.SparseVectorFieldMapper.lambda$static$0(SparseVectorFieldMapper.java:78)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1483)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1441)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseProperties(ObjectMapper.java:325)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseObjectOrDocumentTypeProperties(ObjectMapper.java:235)\n\tat org.elasticsearch.index.mapper.RootObjectMapper.parse(RootObjectMapper.java:411)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:103)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:97)\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:398)\n\t... 23 more\n}], type=mapper_parsing_exception, reason=Failed to parse mapping: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions., caused_by={type=illegal_argument_exception, reason=The [sparse_vector] field type is not supported from 8.0 to 8.10 versions., stack_trace=java.lang.IllegalArgumentException: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.vectors.SparseVectorFieldMapper.lambda$static$0(SparseVectorFieldMapper.java:78)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1483)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1441)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseProperties(ObjectMapper.java:325)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseObjectOrDocumentTypeProperties(ObjectMapper.java:235)\n\tat org.elasticsearch.index.mapper.RootObjectMapper.parse(RootObjectMapper.java:411)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:103)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:97)\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:398)\n\tat org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.updateIndexMappingsAndBuildSortOrder(MetadataCreateIndexService.java:1339)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.lambda$applyCreateIndexWithTemporaryService$2(MetadataCreateIndexService.java:472)\n\tat org.elasticsearch.indices.IndicesService.withTempIndexService(IndicesService.java:731)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexWithTemporaryService(MetadataCreateIndexService.java:470)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequestWithV1Templates(MetadataCreateIndexService.java:593)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequest(MetadataCreateIndexService.java:420)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService$1.execute(MetadataCreateIndexService.java:299)\n\tat org.elasticsearch.cluster.service.MasterService$UnbatchedExecutor.execute(MasterService.java:550)\n\tat org.elasticsearch.cluster.service.MasterService.innerExecuteTasks(MasterService.java:1039)\n\tat org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:1004)\n\tat org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:232)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.lambda$run$2(MasterService.java:1626)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.run(MasterService.java:1623)\n\tat org.elasticsearch.cluster.service.MasterService$5.lambda$doRun$0(MasterService.java:1237)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$5.doRun(MasterService.java:1216)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1623)\n}, stack_trace=org.elasticsearch.index.mapper.MapperParsingException: Failed to parse mapping: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:400)\n\tat org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.updateIndexMappingsAndBuildSortOrder(MetadataCreateIndexService.java:1339)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.lambda$applyCreateIndexWithTemporaryService$2(MetadataCreateIndexService.java:472)\n\tat org.elasticsearch.indices.IndicesService.withTempIndexService(IndicesService.java:731)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexWithTemporaryService(MetadataCreateIndexService.java:470)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequestWithV1Templates(MetadataCreateIndexService.java:593)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService.applyCreateIndexRequest(MetadataCreateIndexService.java:420)\n\tat org.elasticsearch.cluster.metadata.MetadataCreateIndexService$1.execute(MetadataCreateIndexService.java:299)\n\tat org.elasticsearch.cluster.service.MasterService$UnbatchedExecutor.execute(MasterService.java:550)\n\tat org.elasticsearch.cluster.service.MasterService.innerExecuteTasks(MasterService.java:1039)\n\tat org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:1004)\n\tat org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:232)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.lambda$run$2(MasterService.java:1626)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.run(MasterService.java:1623)\n\tat org.elasticsearch.cluster.service.MasterService$5.lambda$doRun$0(MasterService.java:1237)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:368)\n\tat org.elasticsearch.cluster.service.MasterService$5.doRun(MasterService.java:1216)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalArgumentException: The [sparse_vector] field type is not supported from 8.0 to 8.10 versions.\n\tat org.elasticsearch.index.mapper.vectors.SparseVectorFieldMapper.lambda$static$0(SparseVectorFieldMapper.java:78)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1483)\n\tat org.elasticsearch.index.mapper.FieldMapper$TypeParser.parse(FieldMapper.java:1441)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseProperties(ObjectMapper.java:325)\n\tat org.elasticsearch.index.mapper.ObjectMapper$TypeParser.parseObjectOrDocumentTypeProperties(ObjectMapper.java:235)\n\tat org.elasticsearch.index.mapper.RootObjectMapper.parse(RootObjectMapper.java:411)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:103)\n\tat org.elasticsearch.index.mapper.MappingParser.parse(MappingParser.java:97)\n\tat org.elasticsearch.index.mapper.MapperService.parseMapping(MapperService.java:398)\n\t... 23 more\n}"

    at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
    at org.junit.Assert.assertThat(Assert.java:956)
    at org.elasticsearch.test.rest.yaml.section.DoSection.execute(DoSection.java:386)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:562)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:534)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.lang.reflect.Method.invoke(Method.java:578)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
    at java.lang.Thread.run(Thread.java:1623)

```

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/90_sparse_vector.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/90_sparse_vector.yml
@@ -125,7 +125,7 @@
       version: " - 7.99.99, 8.11.0 - "
       reason: "sparse_vector field type not supported in 8.x until 8.11.0"
   - do:
-      catch: /The \[sparse_vector\] field type is no longer supported/
+      catch: /The \[sparse_vector\] field type .* supported/
       indices.create:
         index: test
         body:

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/90_sparse_vector.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/90_sparse_vector.yml
@@ -122,10 +122,8 @@
 ---
 "Sparse vector in 8.x":
   - skip:
-      # version: " - 7.99.99, 8.11.0 - "
-      # reason: "sparse_vector field type not supported in 8.x until 8.11.0"
-      version: "all"
-      reason: "https://github.com/elastic/elasticsearch/issues/99318"
+      version: " - 7.99.99, 8.11.0 - "
+      reason: "sparse_vector field type not supported in 8.x until 8.11.0"
   - do:
       catch: /The \[sparse_vector\] field type .* supported/
       indices.create:

